### DISCUSSION TITLE
Crashing after windows updates.

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#1231] [General]
I know the dev wants us to use discussions instead of the issues area but about 20 minutes ago explorerpatcher was working. I did an update to windows with the update "2023-03 Cumulative Update Preview for Windows 11 Version 22H2 for x64-based Systems (KB5023778)" and now it just loop crashes until I get [This error](https://imgur.com/4gE2Xq2) again, this did not happen until I did that cumulative update.

**Comment by m-brooks** [comment]
How can this issue have no traction?!? Not only is ExplorerPatcher unusable with the 22H2 release of Windows 11 (I have experienced the issue described above on two PCs now), but it makes Windows unusable. I had to restart in command-prompt-only safe mode in order to be able to uninstall ExplorerPatcher. Now I am once again stuck with the W11 Explorer on both machines. Is nobody looking at this?

**Comment by pyrates999** [comment]
I think it's because preview updates do not have symbols to download.  And explorer patcher needs the symbols downloaded to work.  So try uninstalling that preview update and then uninstall and then reinstall explorer patcher.  Make sure you see a notification that symbols have been downloaded.
